.travis.yml

```diff
@@ -47,14 +47,13 @@ before_install:
       sudo pip install cpp-coveralls requests[security];
     else
       brew update; brew update;
-      brew install glew wxwidgets llvm36;
+      brew install glew wxwidgets;
     fi;
 
 before_script:
- - git submodule update --init asmjit ffmpeg
+ - git submodule update --init asmjit ffmpeg llvm
```

Comment:
@Zangetsu38 What? Why are you adding it as a submodule?! We have already [added it as a package](https://github.com/RPCS3/rpcs3/pull/1337/files#diff-354f30a63fb0907d4ad57269548329e3R77).